packages/svelte/src/compiler/utils/ast.js

```diff
@@ -270,6 +272,9 @@ function _extract_paths(assignments = [], param, expression, update_expression)
  * The Acorn TS plugin defines `foo!` as a `TSNonNullExpression` node, and
  * `foo as Bar` as a `TSAsExpression` node. This function unwraps those.
  *
+ * We can't just remove the typescript AST nodes in the parser stage because subsequent
+ * parsing would fail, since AST start/end nodes would point at the wrong positions.
```

Comment:
I tried removing all AST nodes right after parsing in a post-process step, but it failed because of this - if you have an idea how to solve this, I'm all ears

Reply:
That's for removing it during parsing right? We could do one pass after the whole file is parsed.

Reply:
Mhm maybe. I removed it after an expression/the script tag was parsed, but doing it after the whole thing is parsed could maybe work.. would you mind trying it out?